reactor-core/src/test/java/reactor/core/publisher/HooksTestStaticInit.java

```diff
@@ -36,6 +36,8 @@
 import org.junit.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.Matchers.is;
```

Comment:
could you rewrite to use pure `AssertJ` instead? it as an `Assumptions` class that you can use for assumptions.

Reply:
Done. In fact as a die-hard Spock user I haven't used `Assumptions` from AssertJ before, but it looks ok (of course not so nice as `@Require` or `@IgnoreIf` in Spock ;) ).